R/convert.R

```diff
@@ -48,21 +51,29 @@ choose_lookup <- function(frm, to, species = "human") {
 #' @param df Dataframe containing TCR gene names.
 #' @param frm A string, the input format of TCR data. Must be one of "tenx", "adaptive", "adaptivev2", or "imgt".
 #' @param frm_cols A character vector, the custom column names to use.
+#' @param verbose A boolean, whether to show messages.
 #'
 #' @return A character vector, column names to use.
-which_frm_cols <- function(df, frm, frm_cols = NULL) {
+#' @export
+#' @keywords internal
+#' @examples
+#' tcr_file <- get_example_path('tenx.csv')
+#' df <- read.csv(tcr_file)
+#' which_frm_cols(df, 'tenx')
+which_frm_cols <- function(df, frm, frm_cols = NULL, verbose) {
```

Comment:
same here about default value and placement

Reply:
Fixed in e968cf790898e678da8d58d9a37837b257bb09a1.